bin/i18n/resources/pegasus/mobile/sync_in.rb

```diff
@@ -1,34 +1,20 @@
 #!/usr/bin/env ruby
 
-require 'fileutils'
-
 require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_in_base'
 require_relative '../mobile'
 
 module I18n
   module Resources
     module Pegasus
       module Mobile
-        class SyncIn
-          def self.perform
-            new.execute
-          end
-
-          def execute
-            progress_bar.start
+        class SyncIn < I18n::Utils::SyncInBase
+          ORIGINAL_I18n_FILE_PATH = File.join(ORIGIN_I18N_DIR_PATH, 'en-US.yml').freeze
```

Comment:
There's just a small typo here 
```suggestion
          ORIGINAL_I18N_FILE_PATH = File.join(ORIGIN_I18N_DIR_PATH, 'en-US.yml').freeze
```